scripts/committee.ts

```diff
@@ -101,21 +101,35 @@ const app = command({
     let lastBlockFiltered = zkv_descriptor.block_number_before_zkvote_deploy;
     const intfc = zkv.interface;
     let proposalIdToEndBlock: Map<number, number> = new Map<number, number>();
+    let onStartup = true;
     while (true) {
       // Query for new setup proposal events
       const currentBlockNumber = await provider.getBlockNumber();
       if (currentBlockNumber > lastBlockFiltered) {
-        const filter: Filter = zkv.filters.SetupVote();
-        filter.fromBlock = lastBlockFiltered + 1;
-        filter.toBlock = currentBlockNumber;
-        const logs = await provider.getLogs(filter);
+        const setupVoteFilter: Filter = zkv.filters.SetupVote();
+        setupVoteFilter.fromBlock = lastBlockFiltered + 1;
+        setupVoteFilter.toBlock = currentBlockNumber;
+        const logs = await provider.getLogs(setupVoteFilter);
         lastBlockFiltered = currentBlockNumber;
         for (const log of logs) {
           const parsedLog = intfc.parseLog(log);
           const proposalId = parsedLog.args.proposalId;
           const endBlock = parsedLog.args.endBlock;
           proposalIdToEndBlock.set(proposalId, endBlock);
         }
+        // Upon restarting the tallier, we need to check if there are any proposals that have been setup but already tallied. If so, we delete them from our map, since the proposals are no longer active.
+        if (onStartup) {
```

Comment:
I think it might be safe to do this every loop.  Strictly speaking we can probably get away with only doing it on the first iteration, but it might make the code a bit more bullet proof.